### PR TITLE
fix: footer correct position

### PR DESCRIPTION
**Pull request by lamartire**
fix: styles building perfomance fix

### CODE REVIEW

**Comment by lamartire** [comment]
@ifiri ready for review

File: pages/blog/index.vue [hunk 1]
```diff
@@ -1,69 +1,76 @@
...
-                <template slot="title"> {{ p.title }}</template>
-                {{ p.subhead }}
-              </news-card>
+                <news-card v-else :slug="p.title_slug">
+                  <template v-if="p.author" slot="author">
+                    {{ p.author }}
+                  </template>
+                  <template slot="title">
+                    {{ p.title }}</template
+                  >
```
Comment (ifiri) [inline pages/blog/index.vue:49]: strange layout formatting

File: components/PageTemplate.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<template>
```
Comment (dudiq) [inline components/PageTemplate.vue:1]: maybe move `components/PageTemplate.vue` to `layouts` folder?

Comment (lamartire) [reply components/PageTemplate.vue:1]: This is not layout

**Comment by dudiq** [comment]
@kazurus approved

**Pull request merged**